test/integration/native/native_test.go

```diff
@@ -260,9 +262,14 @@ func TestCPUProfiler(t *testing.T) {
 		},
 		{
 			name: "mixed mode unwinding",
-			program: func() (int, func()) {
-				// Test unwinding JIT without frame pointers in the AoT code.
+			program: func(t *testing.T) (int, func()) {
+				t.Helper()
 				// TODO(sylfrena): Remove if condition once toy jit is added for arm64
+				if arch == integration.Arm64 {
+					t.Skip("arm64 toy jit support unimplemented")
```

Comment:
<img src="https://media4.giphy.com/media/u5sgL5pks5JXKHcVZo/giphy.gif"/>

Reply:
Gotta walk before we can run!